test/measures/measures-data-spec.js

```diff
@@ -21,6 +21,40 @@ describe('measures data json', function() {
         .filter(x => requiredAttestationIdsSet.has(x)));
       assert.equal(intersection.size, requiredAttestationIdsSet.size);
     });
+    it('should not have substitutes', () => {
+      requiredAttestationIdsSet.forEach(measureId => {
+        var measure = measuresData.find(m => m.measureId == 'ACI_INFBLO_1');
+        assert.isTrue( _.isEmpty(measure.substitutes));
+      })
+    });
+
+  });
+
+  describe('ACI measures have proper substitutions', () => {
+
+    it('ACI_PHCDRR_1 should be in performanceBonus reporting category', () => {
+      var measure = measuresData.find(m => m.measureId == 'ACI_PHCDRR_1');
+      assert.equal(measure.reportingCategory, 'performanceBonus');
+    });
+    it('ACI_TRANS_PHCDRR_2 should contain correct substitutes', () => {
+      var measure = measuresData.find(m => m.measureId == 'ACI_TRANS_PHCDRR_2');
+      assert.deepEqual(measure.substitutes, ['ACI_PHCDRR_2']);
+    });
+
+    it('should have proper metadata on all measures', () => {
+      var actual = require('../../util/aci-measure-relations.json');
```

Comment:
it's good practice to only require at the top of a file, in the spirit of how es6 import statements work